core/invoker/src/main/resources/application.conf

```diff
@@ -69,6 +69,8 @@ whisk {
   container-factory.container-args {
     network: bridge
     dns-servers: []
+    dns-search: []
```

Comment:
May be add a link in comment here to https://docs.docker.com/config/containers/container-networking/#dns-services

Reply:
Thanks.  Added link.